pyproject.toml

```
@@ -44,7 +44,8 @@ dynamic = ["version"]
 all = [
     "contextily",
     "geopandas",
-    "ipython"
+    "ipython",
+    "rioxarray"
```

Adding a trailing comma following [PEP 8](https://peps.python.org/pep-0008/#when-to-use-trailing-commas), although it's TOML not Python.
```suggestion
    "rioxarray",
```